code/style.css

```diff
@@ -1,4 +1,194 @@
-html {
-  background: #0025ff;
-  color: #fff;
-}
+        html {
+            position: relative;
+            min-height: 100%;
+        }
```

Comment:
No need to indent the CSS as you have done all the way through your stylesheet. It's enough to indent the property values inside the curly brackets 🕺